tests/shared/robot.spec.js

```diff
@@ -131,16 +131,16 @@ describe('Shared: Robot', function() {
         data: {
           message: 'foo',
           code: 'MYERROR',
-          stacktrace: error.stack
+          stack: error.stack
         }
       });
     });
 
     it('should handle a string error', function() {
```

Comment:
Oh right, I've just double-checked and I can now see `.getTitle` and `.getDescription` do support being passed `String`s (with `.getDescription` simply returning `''` if passed a `String`).